diff --git a/third_party/zlib/BUILD.gn b/third_party/zlib/BUILD.gn
--- a/third_party/zlib/BUILD.gn
+++ b/third_party/zlib/BUILD.gn
@@ -88,7 +88,7 @@ if (use_arm_neon_optimizations) {
       defines += [ "ARMV8_OS_LINUX" ]
     } else if (is_mac) {
       defines += [ "ARMV8_OS_MACOS" ]
-    } else if (is_ios) {
+    } else if (is_ios || target_os == "xros") {
       defines += [ "ARMV8_OS_IOS" ]
     } else if (is_win) {
       defines += [ "ARMV8_OS_WINDOWS" ]
